content/en/docs/Writing policies/generate.md

```diff
@@ -246,7 +246,111 @@ spec:
           foo: bar
 ```
 
-## Generating resources into existing namespaces
+## Generate for Existing resources
+
+With Kyverno 1.7.0+, Kyverno supports the generate existing resources. Generate existing policies are applied in the background which creates target resource in existing matched resources in the cluster.
+They may also optionally be configured to apply upon updates to the policy itself.
+
+{{% alert title="Note" color="warning" %}}
+The proper permissions may need to be granted to Kyverno ServiceAccount. Kubernetes resources may require granting additional privileges to the ClusterRole responsible for the generate behavior. To enable Kyverno to generate these other types, edit the ClusterRole typically named kyverno:generate and add or update the rules to cover the resources and verbs needed.
+{{% /alert %}}
+
+### Generate NetworkPolicy on Existing Namespaces
+
+By default, policy will not be applied on existing trigger resource when it is installed. This behavior can be configured via `generateExistingOnPolicyUpdate` attribute. If you set `generateExistingOnPolicyUpdate` to `true`, Kyverno will generate the target secret resource in existing namespaces on policy CREATE and UPDATE events.
```

Comment:
Should we point out that this feature won't work if `generateExistingOnPolicyUpdate` is not set to `true`?

Reply:
added the `Only if you set ...` part , should be good enough.